### PR TITLE
Field cleanup

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Removing a bunch of the fields that we're no longer going to be using, along with the associated template markup.

**Motivation and Context**
Some of the new fields needed to be added into the acf export, and a bunch of the old ones needed to be removed.

**How Has This Been Tested?**
In dev!

**Types of changes**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dev/acf-export.json [hunk 1]
```diff
@@ -242,43 +242,141 @@
...
-                        "key": "field_5e68f05de0d6f",
-                        "label": "Cluster Lead",
-                        "name": "cluster_lead",
-                        "type": "post_object",
-                        "instructions": "Select the faculty cluster lead.",
+                        "key": "field_5e6b945d0d199",
+                        "label": "Story Source",
+                        "name": "story_source",
+                        "type": "radio",
+                        "instructions": "Select the source of the news story",
```
Comment (cadie) [inline dev/acf-export.json]: Small beans, but, most of the instruction lines have periods at the end. Can we add them to all of them for consistency?

File: dev/acf-export.json [hunk 2]
```diff
@@ -242,43 +242,141 @@
...
+                        "layout": "vertical",
+                        "return_format": "value",
+                        "save_other_choice": 0
+                    },
+                    {
+                        "key": "field_5e6b94bd0d19a",
+                        "label": "Today URL",
+                        "name": "today_url",
+                        "type": "url",
+                        "instructions": "The URL of the Today story",
```
Comment (cadie) [inline dev/acf-export.json]: Missing period

File: dev/acf-export.json [hunk 3]
```diff
@@ -242,43 +242,141 @@
...
+                        },
+                        "default_value": "",
+                        "placeholder": ""
+                    },
+                    {
+                        "key": "field_5e6b94de0d19b",
+                        "label": "Pegasus URL",
+                        "name": "pegasus_url",
+                        "type": "url",
+                        "instructions": "The URL of the Pegasus Magazine story",
```
Comment (cadie) [inline dev/acf-export.json]: Missing period

File: dev/acf-export.json [hunk 4]
```diff
@@ -324,127 +559,174 @@
...
-                        "label": "URL",
-                        "name": "college_url",
-                        "type": "url",
-                        "instructions": "The homepage of the college or center.",
-                        "required": 0,
+                        "key": "field_5e6b981de958a",
+                        "label": "Program Name",
+                        "name": "program_name",
+                        "type": "text",
+                        "instructions": "Enter the program name",
```
Comment (cadie) [inline dev/acf-export.json]: Missing period

File: dev/acf-export.json [hunk 5]
```diff
@@ -571,35 +878,60 @@
...
-                "return_format": "object"
+                "placement": "top",
+                "endpoint": 0
+            },
+            {
+                "key": "field_5e6a6dfeedd8e",
+                "label": "Related Social",
+                "name": "cluster_related_tweets",
+                "type": "repeater",
+                "instructions": "Add the URLs to relevant tweet or facebook posts",
```
Comment (cadie) [inline dev/acf-export.json]: Missing period

File: dev/acf-export.json (hunk 1 above)
Comment (jmbarne3) [reply dev/acf-export.json]: Small beans corrected!

**Pull request merged**